recipes/prgenv-gnu/24.11/a100/environments.yaml

```
@@ -0,0 +1,46 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: cuda
+  unify: true
+  specs:
+  - boost +chrono +filesystem +iostreams +mpi +python +regex +serialization +shared +system +timer
+  - cmake
+  - fftw
+  - fmt
+  - gsl
+  - hdf5+cxx+hl+fortran
+  - kokkos +aggressive_vectorization ~alloc_async +cuda_constexpr +cuda_lambda ~cuda_relocatable_device_code ~cuda_uvm cxxstd=17 +openmp +pic +serial +shared +tuning +wrapper
+  - kokkos-kernels +blas +cublas +cusparse +cusolver +execspace_cuda +execspace_openmp +execspace_serial +lapack +memspace_cudaspace +openmp scalars=float,double,complex_float,complex_double +serial +shared +superlu
+  - kokkos-tools +mpi +papi
+  - netlib-scalapack
+  - lua
+  - libtree
+  - lz4
+  - meson
+  - ninja
+  - openblas threads=openmp
+  - osu-micro-benchmarks@5.9
+  - papi ~cuda
+  - python@3.12
+  - zlib-ng
+  # add GPU-specific packages here, for easier comparison with mc version
+  - nccl
+  - nccl-tests
+  - cuda@12.6
+  - aws-ofi-nccl@git.v1.9.2-aws=1.9.2
+  - superlu
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=90
```

```suggestion
  - cuda_arch=86
```
?

---

Sorry,
```suggestion
  - cuda_arch=80
```
actually... I googled too quickly. `80` is also what was set previously in the 23.11 recipe.